flask_security/proxies.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+"""
+flask_security.proxies
+~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+_security = LocalProxy(lambda: current_app.extensions["security"])
```

Comment:
why are you using the `_` as prefix, but then you use it in the package "everywhere". should a `_` prefix normally not indicate to be used only in the file itself?

Reply:
btw the `_security` variable does already exists in the `utils.py` file

Reply:
I am not sure, it's how they have it "upstream" https://github.com/pallets-eco/flask-security/blob/main/flask_security/proxies.py#L13